### PR TITLE
Config updates/fixes to properly configure IntelliSense within the dev and release containers

### PR DESCRIPTION
**Pull request by bettinaheim**

### CODE REVIEW

File: docker/test/opensuse.Dockerfile [hunk 1]
```diff
@@ -14,7 +14,8 @@ ARG pip_install_flags="--user"
 ARG preinstalled_modules="numpy pytest nvidia-cublas-cu11"
 
 ARG DEBIAN_FRONTEND=noninteractive
-RUN zypper --non-interactive up --no-recommends && zypper --non-interactive in --no-recommends \
+RUN zypper clean --all && zypper --non-interactive up --no-recommends \
+    && zypper --non-interactive in --no-recommends \
```
Comment (bmhowe23) [inline docker/test/opensuse.Dockerfile:18]: Was this change intended to fix this error: https://github.com/NVIDIA/cuda-quantum/actions/runs/6865626131/job/18670804127?pr=922 ? If so, it may not have 100% fixed it. That being said - it doesn't seem like this change would have any harm, so it seems file to keep it.

**Review by bmhowe23 — APPROVED** [review]
LGTM, just one minor question below.

File: docker/test/opensuse.Dockerfile (hunk 1 above)
Comment (bettinaheim) [reply docker/test/opensuse.Dockerfile:18]: Yes, and I know. Now it is a different error...
